lightning-invoice/src/lib.rs

```diff
@@ -1292,10 +1290,100 @@ impl<S> Display for SignOrCreationError<S> {
 	}
 }
 
+mod utils {
```

Comment:
I was thinking this could be in its own file as well for code cleanliness.

Reply:
Probably also should be pub :).

Reply:
Probably also should be pub in some way :).